src/Shared/CertificateGeneration/UnixCertificateManager.cs

```diff
@@ -18,7 +18,12 @@ internal UnixCertificateManager(string subject, int version)
     {
     }
 
-    public override bool IsTrusted(X509Certificate2 certificate) => false;
+    public override bool IsTrusted(X509Certificate2 certificate)
+    {
+        using X509Chain chain = new X509Chain();
+        chain.ChainPolicy.RevocationMode = X509RevocationMode.NoCheck;
```

Comment:
Why disable revocation checking?  Perf?

Reply:
`X509Chain` uses `Online` as the default and `SocketsHttpHandler` uses `NoCheck` as the default.

If we don't set this to `NoCheck` then verification fails because the local certificates have no online revocation list to check against.